test/index.js

```diff
@@ -1,81 +1,97 @@
 'use strict';
 
-var assert = require('assert');
-var vm = require('vm');
+const assert = require ('assert');
+const vm = require ('vm');
 
-var Z = require('sanctuary-type-classes');
-var Z$version = require('sanctuary-type-classes/package.json').version;
-var type = require('sanctuary-type-identifiers');
+const Z = require ('sanctuary-type-classes');
+const Z$version = require ('sanctuary-type-classes/package.json') .version;
+const type = require ('sanctuary-type-identifiers');
 
-var $ = require('..');
+const $ = require ('..');
 
 
-//  always :: a -> () -> a
-function always(x) { return function() { return x; }; }
+//    curry2 :: ((a, b) -> c) -> a -> b -> c
+const curry2 = f => x => y => f (x, y);
 
-//  eq :: (a, b) -> Undefined !
-function eq(actual, expected) {
-  assert.strictEqual(arguments.length, eq.length);
-  assert.strictEqual(Z.toString(actual), Z.toString(expected));
-  assert.strictEqual(Z.equals(actual, expected), true);
-}
-
-//  identity :: a -> a
-function identity(x) { return x; }
-
-//  length :: { length :: a } -> a
-function length(x) { return x.length; }
+//    curry3 :: ((a, b, c) -> d) -> a -> b -> c -> d
+const curry3 = f => x => y => z => f (x, y, z);
 
-//  notImplemented :: () -> Undefined !
-function notImplemented() { throw new Error('Not implemented'); }
-
-//  throws :: (Function, TypeRep a, String) -> Undefined
-function throws(f, type, message) {
-  assert.throws(f, function(err) {
-    return err.constructor === type && err.message === message;
-  });
+//    eq :: (a, b) -> Undefined !
+function eq(actual, expected) {
+  assert.strictEqual (arguments.length, eq.length);
```

Comment:
Curious that you use the new style in the examples and the tests, but not the actual source.

Reply:
I can imagine reformatting the source code too, but I'd prefer to do so in a separate pull request to avoid obfuscating the significant code changes.